tests/opensource_demo_orangehrmlive/admin/employment_statuses/test_put_employment_status.py

```diff
@@ -26,6 +26,7 @@ def test_BYT_T101_Actualizar_un_estado_de_empleado_y_guardar_con_nombre_valido(s
 
     response = requests.put(url, headers=header, data=payload)
     assert response.status_code == 200
```

Comment:
Reutiliza el assert que tenemos en common assertions
```suggestion
    assert_status_code(response, expected_status=200)
```